fastfield_codecs/src/lib.rs

```diff
@@ -69,20 +69,20 @@ pub struct FastFieldStats {
 }
 
 impl<'a> FastFieldDataAccess for &'a [u64] {
-    fn get(&self, doc: u32) -> u64 {
-        self[doc as usize]
+    fn get_val(&self, position: u64) -> u64 {
+        self[position as usize]
     }
 }
 
 impl<'a> FastFieldDataAccess for &'a Vec<u64> {
```

Comment:
I suspect this one is useless...

Reply:
because &[u64] exists? type deduction doesn't work in some case properly

Reply:
let's remove it...